chaco/tools/tracking_zoom.py

```diff
@@ -24,10 +24,10 @@ def normal_mouse_wheel(self, event):
         if self.enable_wheel and event.mouse_wheel != 0:
             if event.mouse_wheel > 0:
                 # zoom in
-                zoom = 1.0 / (1.0 + 0.5 * self.wheel_zoom_step)
+                zoom = 2.0 / (1.0 + self.zoom_factor)
             elif event.mouse_wheel < 0:
                 # zoom out
-                zoom = 1.0 + 0.5 * self.wheel_zoom_step
+                zoom = (1.0 + self.zoom_factor) / 2.0
```

Comment:
Note that `self.wheel_zoom_step = self.zoom_factor - 1.0`

Reply:
we don't need the `2.0`, `1.0` anymore on python 3